test/e2e/tests/tokens/nft/erc1155-interaction.spec.js

```diff
@@ -65,6 +75,45 @@ describe('ERC1155 NFTs testdapp interaction', function () {
           true,
```

Comment:
nit: given that we are editing this test, could we remove the extra assert and just do this:

```
await driver.waitForSelector({
          css: '[data-testid="activity-list-item-action"]',
          text: 'Deposit',
});
```